.github/workflows/website.yml

```diff
@@ -0,0 +1,30 @@
+name: Website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'website/**'
```

Comment:
We could also run this workflow manually
🔗 https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow

```yml
on: workflow_dispatch
```